contrib/docker-compose/standalone/mksite.bash

```diff
@@ -117,4 +117,18 @@ sed -E -i -e '/^(PROMETHEUS_HOST_PORT|CERTBOT_HOST_PORT|HAP_DHPARAM_FILE|HAP_CON
 echo "# MIGRATION_3_DONE (DO NOT REMOVE THIS LINE!)" >> "$HERE/.env"
 fi
 
+if ! grep -q MIGRATION_4_DONE "$HERE/.env"; then
+# Migration 4: new haproxy configuration (additional)
+
+cat >>"$HERE/.env" <<EOF
+
+# Expose the admin socket to the host
+HAP_ADMIN_HOST_PORT=127.0.0.1:1396
```

Comment:
Do we need a migration 4? These are already the defaults in the docker-compose.yml. We can add commented out lines in the main section for documentation purposes but it doesn't actually matter if people don't run mksite...

Reply:
Would it be better to maintain (and copy from) a full sample config file, rather than dynamically generate the live one? That way there will always be an up to date reference. We'd still have to maintain migrations though...

Reply:
I removed the migration since it's optional.